### ISSUE TITLE
Have a changelog file -

### ISSUE THREAD

**Issue by shirishag75** [HelloZeroNet/ZeroNet#606]
This is being built on https://github.com/HelloZeroNet/ZeroNet/issues/605 . As of now there is no way to know what changes are being made besides downloading via git and then trying to compile rather than the other way around. It would be nice if a changelog file is there and it has commit messages in it. Would help in packaging from unix distributions as well.

**Comment by ashleypt** [comment]
I'll take this task on! :)

**Comment by shirishag75** [comment]
probably just formatted git log would be good enough. It could be as simple as -

Using nano's changelog.gz from /usr/share/doc/nano -

```
2016-03-31  Benno Schulenberg  <bensberg@justemail.net>
        * src/text.c (do_int_spell_fix): Replace a fake 'while', delete a
        redundant 'if', rename a variable, and adjust whitespace.

2016-03-30  Benno Schulenberg  <bensberg@justemail.net>
        * src/search.c (do_replace_loop, go_looking, findnextstr): Report
        "Cancelled" instead of "Not found" when the user aborts a replace
        that is taking too long.  This fixes Savannah bug #47439.
        * src/winio.c (do_replace_highlight): Rename this to 'spotlight',
        for clarity, and for contrast with 'do_replace/do_replace_loop'.
        * src/winio.c (spotlight): Rename a variable for clarity.
        * src/files.c (input_tab), src/prompt.c (get_prompt_string):
        Rename a variable to better indicate booleanness.
        * src/text.c (do_int_speller): Unwrap a few lines.

It basically needs a list of the changes done between various versions. Bonus points if versioning is also mentioned (which is not seen in the above changelog.gz) and the author's name with e-mail id (if he doesn't want to obfuscate it ) .  Makes it easier to see what the journey has been. 


```

**Comment by HelloZeroNet** [comment]
A manually written changelog has benn added: https://github.com/HelloZeroNet/ZeroNet/blob/master/CHANGELOG.md
